### PR TITLE
test_setup: check if set hugepages on target node successfully

### PR DESCRIPTION
**Pull request by huangyum**
ID:  1456123
Signed-off-by: Yumei Huang <yuhuang@redhat.com>

### CODE REVIEW

**Comment by huangyum** [comment]
(1/2) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.hotplug_memory.after.guest_migration.hotplug.backend_file.policy_default.two: PASS (342.16 s)
 (2/2) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.hotplug_memory.after.guest_migration.hotplug.backend_file.policy_bind.two: PASS (246.88 s)

**Comment by vivianQizhu** [comment]
@huangyum Please help state the issue and the solution in the commit log. Thanks.

**Comment by vivianQizhu** [comment]
@MiriamDeng Please help review, thanks.

**Comment by MiriamDeng** [comment]
Sure,I need to check it on ppc platform as soon as I got the log,thanks.

**Comment by huangyum** [comment]
@vivianQizhu Updated the commit message, thanks

File: virttest/test_setup.py [hunk 1]
```diff
@@ -452,7 +452,7 @@ def __init__(self, params):
                         # if target_hugepages is not divisible by node_len
                         # add remainder it to first node
                         num += self.target_hugepages % self.node_len
-                self.target_node_num[node] = num
+                self.target_node_num[node] = int(num)
```
Comment (luckyh) [inline virttest/test_setup.py]: Also requires to update line 450 to be:
```python
num = self.target_hugepages // self.node_len
```

Comment (huangyum) [reply virttest/test_setup.py]: Thanks, updated.

**Comment by luckyh** [comment]
No problem, thanks all.

**Pull request merged**